client/extensions/woocommerce/app/settings/shipping/shipping-zone/shipping-zone-locations.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import ExtendedHeader from 'woocommerce/components/extended-header';
+import List from 'woocommerce/components/list/list';
+
+const ShippingZoneLocations = ( { translate } ) => {
+	const renderContent = () => {
+		return null;
+	};
+
+	return (
+		<div className="shipping-zone__locations">
+			<ExtendedHeader
+				label={ translate( 'Zone locations' ) }
+				description={ translate( 'Add locations that you want to share shipping methods' ) } >
+				<Button>{ translate( 'Add location' ) }</Button>
```

Comment:
Hi! I've found a possible matching string that has already been translated **46** times:
`translate( 'Add Location' )` *ES Score: 15*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).